components/event-service/event/events.go

```diff
@@ -177,6 +178,18 @@ func (svc Events) getClient(handlerType string) (EventHandlerClient, error) {
 			return nil, errors.New("CallHandler could not obtain NewProjectsClient")
 		}
 		return authzProjectsClient, nil
+	} else if handlerType == config.EVENT_FEED {
```

Comment:
[nit] I know you're just adding one, but wouldn't this make sense as a `switch` statement?

Reply:
I don't see why not to use a `switch` I will try it out.